### PR TITLE
Ian/feature/voice

### PR DESCRIPTION
**Pull request by ianchalk**
- added play.js and disconnect.js
- added sound files in \assets\sounds
- added bot intents in index.js
- 3 dependencies: @discordjs/opus. @discordjs/voice, libsodium-wrappers

### CODE REVIEW

File: src/commands/voice/disconnect.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    // Check if user is connected to a voice channel
+    if (!userVoiceChannel) {
+      return await interaction.reply(
+        "You must be in a voice channel to use the disconnect command."
+      );
+    }
+
+    // Get the bot voice connection and destroy it
+    const connection = getVoiceConnection(interaction.guild.id);
```
Comment (michaelFarrier) [inline src/commands/voice/disconnect.js]: Use `userVoiceChannel.guild.id` as shown in this [link](https://discordjs.guide/voice/voice-connections.html#creation).

File: src/commands/voice/play.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        filePath = `../../assets/sounds/general/bingBong.mp3`;
+
+        break;
+
+      default:
+        return await interaction.reply("Error: could not find audio category");
+    }
+
+    // Get the bot voice connection and destroy it
+    let connection = getVoiceConnection(interaction.guild.id);
```
Comment (michaelFarrier) [inline src/commands/voice/play.js]: Similar to disconnect comment, use `channel.guild.id`

File: src/commands/voice/play.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    const player = createAudioPlayer();
+
+    let resource = createAudioResource(join(__dirname, `${filePath}`), {
+      inlineVolume: true,
+    });
+    resource.volume.setVolume(0.5);
+
+    connection.subscribe(player);
```
Comment (michaelFarrier) [inline src/commands/voice/play.js]: Please add a check for subscription as shown [here](https://discordjs.guide/voice/voice-connections.html#playing-audio)

File: src/index.js [hunk 4]
```diff
@@ -7,46 +7,54 @@ const mongodb = require("./database/mongo");
...
+  ],
+});
 
 // COMMAND HANDLER
 client.commands = new Collection();
 const foldersPath = path.join(__dirname, "commands");
 const commandFolders = fs.readdirSync(foldersPath);
 
 for (const folder of commandFolders) {
-	const commandsPath = path.join(foldersPath, folder);
```
Comment (michaelFarrier) [inline src/index.js:18]: Can you check your prettier.js settings or vscode settings to make sure that these are tabbed or 4 spaces long? Not sure why these are shifting around.

**Review by michaelFarrier — CHANGES_REQUESTED** [review]
Please review the comments and re-submit.

File: src/commands/voice/play.js (hunk 3 above)
Comment (ianchalk) [reply src/commands/voice/play.js]: How long do you want the subscription to last? What's the time limit on audio files? I set it to 5 seconds for now.

File: src/commands/voice/disconnect.js (hunk 1 above)
Comment (ianchalk) [reply src/commands/voice/disconnect.js]: Corrected.

File: src/commands/voice/play.js (hunk 2 above)
Comment (ianchalk) [reply src/commands/voice/play.js]: Corrected.

File: src/index.js (hunk 4 above)
Comment (ianchalk) [reply src/index.js:18]: image
It looks properly tabbed to me. Letme check the tab spaces

Comment (ianchalk) [reply src/index.js:18]: Tab width was set to 2 spaces. I changed it to 4

**Pull request closed without merging**